cluster/addons.go

```diff
@@ -4,12 +4,15 @@ import (
 	"bytes"
 	"context"
 	"fmt"
-	rkeData "github.com/rancher/kontainer-driver-metadata/rke/templates"
-	"github.com/rancher/rke/templates"
 	"os"
 	"os/exec"
 	"time"
 
+	rkeData "github.com/rancher/kontainer-driver-metadata/rke/templates"
```

Comment:
Please re-arrange the imports. [standard golang imports + newline + rest of the stuff] and run goimports

Reply:
done